helm_deploy/hmpps-incentives-ui/values.yaml

```diff
@@ -56,8 +56,6 @@ generic-service:
 
   allowlist:
     global-protect: "35.176.93.186/32"
-    moj-official-tgw-prod: "51.149.250.0/24"
```

Comment:
well spotted! Maybe for the future we should aim at trying to have lists like this in alphabetical order as it should help spot things like this more easily ☺️

Reply:
i didn't compare the ranges themselves, however. it's possible they overlap somewhere